Makefile

```diff
@@ -1,27 +1,29 @@
 ### This is the Terraform-generated header for the timdex-pipeline-lambads Makefile ###
 SHELL=/bin/bash
 DATETIME:=$(shell date -u +%Y%m%dT%H%M%SZ)
-ECR:=222053980223.dkr.ecr.us-east-1.amazonaws.com
-ECR_NAME:=timdex-pipeline-lambdas-dev
-ECR_URL:=222053980223.dkr.ecr.us-east-1.amazonaws.com/timdex-pipeline-lambdas-dev
-FUNCTION:=timdex-format-dev
+### This is the Terraform-generated header for timdex-pipeline-lambdas-dev
+ECR_NAME_DEV:=timdex-pipeline-lambdas-dev
+ECR_URL_DEV:=222053980223.dkr.ecr.us-east-1.amazonaws.com/timdex-pipeline-lambdas-dev
+FUNCTION_DEV:=timdex-format-dev
 ### End of Terraform-generated header ###
 
-
 help: ## Print this message
 	@awk 'BEGIN { FS = ":.*##"; print "Usage:  make <target>\n\nTargets:" } \
 /^[-_[:alpha:]]+:.?*##/ { printf "  %-15s%s\n", $$1, $$2 }' $(MAKEFILE_LIST)
 
-dist-dev: ## Build docker container
+### Developer Deploy Commands ###
+dist-dev: ## Build docker container (intended for developer-based manual build)
 	docker build --platform linux/amd64 \
-	    -t $(ECR_URL):latest \
-		-t $(ECR_URL):`git describe --always` \
-		-t $(ECR_NAME):latest .
+	    -t $(ECR_URL_DEV):latest \
+		-t $(ECR_URL_DEV):`git describe --always` \
+		-t $(ECR_NAME_DEV):latest .
 
-publish-dev: dist-dev ## Build, tag and push
-	docker login -u AWS -p $$(aws ecr get-login-password --region us-east-1) $(ECR_URL)
-	docker push $(ECR_URL):latest
-	docker push $(ECR_URL):`git describe --always`
+publish-dev: dist-dev ## Build, tag and push (intended for developer-based manual publish)
+	docker login -u AWS -p $$(aws ecr get-login-password --region us-east-1) $(ECR_URL_DEV)
+	docker push $(ECR_URL_DEV):latest
+	docker push $(ECR_URL_DEV):`git describe --always`
 
-update-lambda-dev: ## Updates the lambda with whatever is the most recent image in the ecr
-	aws lambda update-function-code --function-name $(FUNCTION) --image-uri $(ECR_URL):latest
+update-lambda-dev: ## Updates the lambda with whatever is the most recent image in the ecr (intended for developer-based manual update)
+	aws lambda update-function-code \
+		--function-name $(FUNCTION_DEV) \
+		--image-uri $(ECR_URL_DEV):latest
```

Comment:
I don't know the answer to this question: Does a `Makefile` require a blank line at the end? If so, this doesn't have it. If not, all good.

Reply:
I dont believe it does, but i've added one anyway